lib/spaceship/tunes/tunes_client.rb

```diff
@@ -314,10 +314,21 @@ def app_version(app_id, is_live)
 
       # We only support platforms that exist ATM
       platform = platforms.find do |p|
-        ['ios', 'osx'].include? p['platformString']
+        ['ios', 'osx', 'appletvos'].include? p['platformString']
       end
 
-      raise "Could not find platform ios or osx for app #{app_id}" unless platform
+      raise "Could not find platform ios, osx or appletvos for app #{app_id}" unless platform
+
+      # If your app has versions for both iOS and tvOS we will default to returning the iOS version for now.
+      # This is intentional as we need to do more work to support apps that have hybrid versions.
+      if platforms.length > 1
+        platforms.each do |p|
+          if p['platformString'].eql? "ios"
+            platform = p
+            break
+          end
+        end
+      end
```

Comment:
This could be more Ruby like maybe?

```ruby
if platforms.length > 1
  platform = platforms.find do |p|
    p['platformString'] == "ios"
  end
end
```
What do you think?